Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -408,12 +408,12 @@ private extension StripeCardReaderService {
                     /// https://stripe.dev/stripe-terminal-ios/docs/Classes/SCPTerminal.html#/c:objc(cs)SCPTerminal(im)collectPaymentMethod:delegate:completion:
 
                     if underlyingError != .commandCancelled {
-                        print("==== collect payment method was not cancelled. this is an actual error ", underlyingError)
+                        DDLogError("💳 Error: collect payment method \(underlyingError)")
                         promise(.failure(CardReaderServiceError.paymentMethodCollection(underlyingError: underlyingError)))
                     }
 
                     if underlyingError == .commandCancelled {
-                        print("==== collect payment method cancelled. this is an error we ignore ", error)
+                        DDLogError("💳 Warning: collect payment error cancelled. We actively ignore this error \(error)")
```

Comment:
I think this should probably be Info or Warn level, given that we ignore it...

Reply:
Good point! I pushed an update, and enabled auto-merge